.github/workflows/e2e-tests.yml

```diff
@@ -155,17 +155,64 @@ jobs:
       name: Flaky
       tags: "@flaky"
 
+  # NEW: Merge split timings from all E2E test runs
+  merge-split-timings:
+    needs: [e2e-tests]
+    if: ${{ always() }}
```

Comment:
this makes sense for testing, but I don't think we want to update timings on every PR, probably nightly at most